piqueserver/server.py

```diff
@@ -95,8 +96,11 @@ def ensure_dir_exists(filename):
     d = os.path.dirname(filename)
     try:
         os.makedirs(d)
-    except FileExistsError:
-        pass
+    except OSError as e:
+        if e.errno == errno.EEXIST:
+            pass
+        else:
+            raise
```

Comment:
should be `raise e`

Reply:
@swalladge fixed the issues in latest commit can you re-review it?

Reply:
@godwhoa looks good to me. 👍